src/bot.py

```diff
@@ -255,71 +404,117 @@ async def update(ctx):
+@client.command(name="duis", pass_context=True)
 async def duis(ctx):
-    await ctx.send(EasterEggs.Duis())
+    await ctx.reply(EasterEggs.Duis())
+
+
+@client.command(name="furry", pass_context=True)
+async def furry(ctx):
+    await ctx.reply(EasterEggs.Furry())
+
+
+@client.command(name="don", pass_context=True)
+async def don(ctx):
+    await ctx.reply(EasterEggs.Don())
+
+
+@client.command(name="daffy", pass_context=True)
+async def daffy(ctx):
+    await ctx.reply(EasterEggs.Daffy())
+
+
+@client.command(name="giddy", pass_context=True)
+async def giddy(ctx):
+    await ctx.reply(EasterEggs.Giddy())
+
+
+@client.command(name="nodought", pass_context=True)
+async def nodought(ctx):
+    await ctx.reply(EasterEggs.NoDought())
 
 
-@client.command(name='normq', pass_context=True)
+@client.command(name="coolio", pass_context=True)
+async def coolio(ctx):
+    await ctx.reply(EasterEggs.Coolio())
+
+
+@client.command(name="normq", pass_context=True)
 async def normq(ctx):
-    messages: List[str or Embed] = EasterEggs.NormQ()
-    for msg in messages:
-        if (isinstance(msg, Embed)):
-            await ctx.send(embed=msg)
-        else:
-            await ctx.send(msg)
+    blueTeam, _ = Queue.getTeamList()
+    if (len(blueTeam) >= 1):
+        await ctx.send("Duis says I am not supposed to queue, but I don't listen to players worse than me...")
+        await ctx.send("!q")
+        await sendMessage(ctx, EasterEggs.NormQ(), "active")
+    else:
+        await ctx.send("Duis says I am not supposed to queue, but I don't listen to players worse than me...")
+        await ctx.send("!q")
+        await sendMessage(ctx, EasterEggs.NormQ(),
+                          "queue" if len(blueTeam) == 0 and
+                          Queue.getQueueLength() != 6
+                          else "popped")
 
 
-@client.command(name='teams', aliases=['uncc'], pass_context=True)
+@client.command(name="teams", aliases=["uncc"], pass_context=True)
 async def teams(ctx):
-    await ctx.send(EasterEggs.Teams())
+    await ctx.reply(EasterEggs.Teams())
 
 
-@client.command(name='8ball', aliases=['norm', 'asknorm', 'eight_ball', 'eightball', '8-ball'], pass_context=True)
+@client.command(name="8ball", aliases=["norm", "asknorm", "eight_ball", "eightball", "8-ball"], pass_context=True)
 async def eight_ball(ctx):
-    await ctx.send(EasterEggs.EightBall(ctx.message.author))
+    await ctx.reply(EasterEggs.EightBall(ctx.message.author))
 
 
-@client.command(name='fuck', aliases=['f', 'frick'], pass_context=True)
+@client.command(name="fuck", aliases=["f", "frick"], pass_context=True)
 async def fuck(ctx):
-    await ctx.send("u")
+    await ctx.reply(EasterEggs.Fuck())
 
 
 @client.command(name="help", pass_context=True)
 async def help(ctx):
     await ctx.send(embed=HelpEmbed())
 
 
+@client.command(name="oops", pass_context=True)
+async def oops(ctx):
+    await ctx.reply("I didn't think the queue would pop...")
```

Comment:
This should live in `EasterEggs.py` in a function named `Oops`.

Reply:
Resolved.